Add Supernotes integration documentation

BredeYabo opened the pull request:
## Summary
- Add documentation for Supernotes, an AI-powered meeting assistant integration
- Includes getting started, workspaces, teams, summaries, settings, and privacy docs
- Added entry to integrations/toc.yml for navigation

## Test plan
- [ ] Verify documentation renders correctly in DocFX
- [ ] Check all internal links work
- [ ] Review content accuracy

digitaldiina commented on integrations/supernotes/toc.yml:
```diff
@@ -0,0 +1,16 @@
+  href: workspaces.md
+- name: Teams
+  href: teams.md
+- name: Before Meeting
```
```suggestion
- name: Before meeting
```

digitaldiina commented on integrations/supernotes/index.md:
```diff
@@ -0,0 +1,82 @@
+keywords: Supernotes, AI, meetings, transcription, summaries, SuperOffice
+author: Brede Kristensen
+date: 12.15.2025
+content_type: overview
```
```suggestion
content_type: concept
```

digitaldiina commented on integrations/supernotes/getting-started.md:
```diff
@@ -0,0 +1,227 @@
+* **Detecting action items and decisions:** automatically identify key outcomes during meetings.
+
+Supernotes is designed to free up time for professional discussions and reduce time spent writing meeting minutes and finding previous decisions. The goal is more effective meetings and higher quality work.
+
```
This is conceptual info already covered in index.md and should be removed from this page. You could replace it with a Related content H2 at the end of this page and link "What is Supernotes and what does it do?" back to index.md

digitaldiina commented on integrations/supernotes/getting-started.md:
```diff
@@ -0,0 +1,227 @@
+| Google Meet       | Fully supported                   |
+| Zoom              | Fully supported                   |
+| Physical meetings | Supported (with recording device) |
+
```
Also covered in index.md. Convert to related content link.

digitaldiina commented on integrations/supernotes/getting-started.md:
```diff
@@ -0,0 +1,227 @@
+
+If Supernotes doesn't automatically join a meeting, you can manually send it:
+
+1. Click **Send Supernotes to meeting** button in the top right corner of the dashboard.
```
```suggestion
1. Click the **Send Supernotes to meeting** button in the upper-right corner of the dashboard.
```

digitaldiina commented on integrations/supernotes/getting-started.md:
```diff
@@ -0,0 +1,227 @@
+1. Open Supernotes on your phone (via web browser).
+2. Log into your account.
+3. Tap the large **Physical Meeting** button on the home screen.
+4. Place your phone in the center of the table.
```
```suggestion
4. Place your phone at the center of the table.
```

digitaldiina commented on integrations/supernotes/workspaces.md:
```diff
@@ -0,0 +1,159 @@
+* **Administrators can manage:** team access and settings.
+* **Usage is tracked collectively:** for billing purposes.
+
+Every Supernotes user must be part of a workspace. This is enforced during onboarding because it's critical for the product to work correctly.
```
```suggestion
Every Supernotes user must be part of a workspace. This is enforced during onboarding.
```
No need to state the reason here since it follows in the next section.

digitaldiina commented on integrations/supernotes/workspaces.md:
```diff
@@ -0,0 +1,159 @@
+> [!TIP]
+> If you see multiple workspace options (common for large organizations), choose the one that matches your actual team or department.
+
+## How do I create a workspace?
```
I recommend you add an anchor here and link to it from the Related content section of getting-started.md

digitaldiina commented on integrations/supernotes/teams.md:
```diff
@@ -0,0 +1,102 @@
+
+More team features will be added in the future, making teams an increasingly powerful way to collaborate.
+
+## How do I create a team?
```
I recommend you add an anchor here and link to it from the Related content section in getting-started.md

digitaldiina commented on integrations/supernotes/before-meeting.md:
```diff
@@ -0,0 +1,91 @@
+> [!NOTE]
+> This feature uses the same "Pre-meeting Briefing" setting as pre-meeting research emails. Both are enabled together in **Settings** > **Profile**.
+
+## How do I enable the note taker for a specific meeting?
```
This overlaps with content in getting-started.md - consider linking instead of repeating (easier to maintain, and this page is about emails)

digitaldiina commented on integrations/supernotes/summaries.md:
```diff
@@ -0,0 +1,236 @@
+* **Next steps:** follow-up items and future plans.
+* **Participants:** who attended the meeting.
+
+The specific sections and format depend on your selected meeting type (e.g., General, Customer Meeting, Internal Meeting, or custom templates you've created).
```
```suggestion
The specific sections and format depend on your selected meeting type (General, Customer Meeting, Internal Meeting, or custom templates you've created).
```

digitaldiina commented on integrations/supernotes/summaries.md:
```diff
@@ -0,0 +1,236 @@
+### Example queries
+
+* "What decisions were made about the budget?"
+* "What did [person] say about [topic]?"
```
Might need to escape the square brackets.

digitaldiina commented on integrations/supernotes/summaries.md:
```diff
@@ -0,0 +1,236 @@
+
+1. Go to the meeting summary page.
+2. Click on the **Details & Recording** tab.
+3. Select **Transcript** from the sub-tabs.
```
We don't use the word "sub-tab"¨- is this a secondary row of tabs within Details & Recording? If **Transcript** is visible at all times after step 2, "Select **Transcript**." is sufficient.

digitaldiina reviewed (changes requested):
Happy new year, @BredeYabo 

I've added some suggestions you can commit directly as well as recommendations for linking, and a few concerns about duplicate content. This is starting to look great!

BredeYabo commented:
> Happy new year, @BredeYabo
> 
> I've added some suggestions you can commit directly as well as recommendations for linking, and a few concerns about duplicate content. This is starting to look great!

Happy new year!

Thanks for the good and detailed feedback. I've tried to accommodate the documentation to fit the overall styling of this repository even more. Removed some of the duplicate documentation, so that it will be easier to make changes in the documentation in the future without making the mistake of not updating everything!
